content/working-groups/learning/minutes/2019.06.11-meeting.md

```diff
@@ -43,6 +43,6 @@ type: docs
 - There is a planned contributing page: https://github.com/rust-lang/www.rust-lang.org/issues/440
 - If someone has time we (WG-learning) should try to push that forward, otherwise, waiting for this to be done...
 
-## adding people rustc-guide toolstate
+## adding people rustc-dev-guide toolstate
```

Comment:
Not sure this is correct, reporting just in case

Reply:
hmmm, not sure what this header means

Reply:
```suggestion
## adding people to the rustc-dev-guide toolstate list
```
probably my fault. toolstate is the system that does linkchecking on the rust-lang/rust repo side (it also checks other tools like miri, clippy, etc for failing tests)